flink-runtime-web/web-dashboard/src/app/pages/job/dataskew/data-skew.component.html

```diff
@@ -0,0 +1,72 @@
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one
+  ~ or more contributor license agreements.  See the NOTICE file
+  ~ distributed with this work for additional information
+  ~ regarding copyright ownership.  The ASF licenses this file
+  ~ to you under the Apache License, Version 2.0 (the
+  ~ "License"); you may not use this file except in compliance
+  ~ with the License.  You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<nz-card nzType="inner" nzTitle="What is Data Skew?">
+  <p>
+    Your Flink job has data skew when a subset of subtasks in any of the operators receives a
+    disproportionate number of records, potentially overloading a subset of task managers while the
+    rest remain idle, leading to inefficient processing and potentially backpressure and other
+    related problems.
+  </p>
+</nz-card>
+
+<nz-card nzType="inner" nzTitle="Data Skew" [nzLoading]="isLoading" [nzExtra]="extraTemplate">
+  <nz-table
+    class="no-border small"
+    [nzSize]="'small'"
+    [nzData]="listOfVerticesAndSkew"
+    [nzFrontPagination]="false"
+    [nzShowPagination]="false"
+  >
+    <tbody>
+      <tr>
+        <th>Vertex</th>
+        <th>Data Skew Percentage</th>
+      </tr>
+      <ng-container *ngFor="let vertexSkew of listOfVerticesAndSkew">
+        <tr>
+          <td>{{ vertexSkew.vertexName }}</td>
+          <td>{{ vertexSkew.skewPct }}%</td>
+        </tr>
+      </ng-container>
+    </tbody>
+  </nz-table>
+</nz-card>
+
+<nz-card nzType="inner" nzTitle="How is Data Skew Calculated?">
+  <p>
+    Data skew is calculated using
+    <a target="_blank" href="https://en.wikipedia.org/wiki/Coefficient_of_variation">
+      Coefficient of Variation (CV)
+    </a>
+    . The
+    <i>numRecordsIn</i>
+    metric is used to calculate the skew across the subtasks of your operators. Therefore the source
+    vertices will not show any data skew, since they do not receive any records. The data skew
+    percentage under the Overview tab shows a more current data skew percentage by using the
+    <i>numrecordsInPerSecond</i>
+    metric and therefore can be different from the skew percentage shown here.
+  </p>
+</nz-card>
+
+<ng-template #extraTemplate>
+  <button nz-button nzType="primary" class="refresh" nzSize="small" (click)="refresh()">
```

Comment:
Can we also add a "lastRefreshedAt" timestamp?

Reply:
I think this might be a good addition, however please note we don't appear to have this in other places e.g. the Checkpoints tab. So it would be slightly inconsistent. 

I can do this in a follow-up PR and discuss if we're OK with the inconsistency in the meantime, or whether we wanna add this to the Checkpoints tab too.